### PR TITLE
new manager option to create admin user with prompts

### PR DESCRIPTION
**Pull request by aharelick**
@sandlerben can you take a look. Having to create a function like this might be the tipping point for me in terms of us possibly thinking about another workflow for when someone starts coding.

### CODE REVIEW

**Comment by aharelick** [comment]
Or do we want to reference something like this from setup_dev because the only reason you should be inserting roles is if you've cleared the database?

**Comment by sandlerben** [comment]
Would you want to move this into the `add_fake_data` command? Three options:
1. People can pass in an email and password to the `add_fake_data` command.
2. Each team can set an admin account which is always created when they call `setup_dev`. For example, for GPCAH, we had an account `ben@ben.com` with password `password`.
3. When `debug=True` we can add an option to the registration form to select your role.

**Comment by sandlerben** [comment]
What do you mean by

> Or do we want to reference something like this from setup_dev because the only reason you should be inserting roles is if you've cleared the database?

**Comment by jondubin** [comment]
We modified user creation so that the first user created is the admin. AFAIK this is a pretty standard practice

**Comment by sandlerben** [comment]
I personally prefer option 3 actually, but that's a good idea Dubin. Let's leave this out of flask-base for now and come together on a solution at the end of the semester.

**Comment by aharelick** [comment]
I'm going to implement option 2. I like option 3 as well, but I don't feel great about having something like this that entrenched in the code when we aren't set on it as a solution for everything.

**Comment by aharelick** [comment]
Can you guys just give it a quick once over before I merge it, thanks.

**Comment by sandlerben** [comment]
Sure. You should have someone from your team review too!

File: manage.py [hunk 1]
```diff
@@ -26,6 +26,21 @@ def make_shell_context():
 manager.add_command('db', MigrateCommand)
 
 
+def create_default_admin(first_name, last_name, email, password):
+    if (User.query.filter_by(email=email).first() is not None):
```
Comment (sandlerben) [inline manage.py]: Nit: parenthesis around statements are not Pythonic

Did flake8 not get mad about this?

File: manage.py [hunk 2]
```diff
@@ -26,6 +26,21 @@ def make_shell_context():
...
+        return
```
Comment (sandlerben) [inline manage.py]: Why don't you move this logic to the `setup_dev` function below. That way we only `create_default_admin` if that account doesn't exist already.

File: manage.py [hunk 3]
```diff
@@ -26,6 +26,21 @@ def make_shell_context():
 manager.add_command('db', MigrateCommand)
 
 
+def create_default_admin(first_name, last_name, email, password):
```
Comment (sandlerben) [inline manage.py]: I feel like this function belongs better in `models/user.py`

We should try to keep this file to just commands.

**Comment by sandlerben** [comment]
Left a few small comments, besides that lgtm

File: manage.py (hunk 1 above)
Comment (aharelick) [inline manage.py]: It's actually not a pep8 thing, just a pycharm thing. But you're right, they are redundant.

Comment (sandlerben) [inline manage.py]: should be, let's amend pep8

Comment (jondubin) [inline manage.py]: Related: more pythonic would be `if not User.query.filter_by(email=email).first():` Whether it's more/less readable, I'll let you decide

**Comment by aharelick** [comment]
Thanks guys, I appreciate it.

File: manage.py [hunk 4]
```diff
@@ -64,6 +64,13 @@ def setup_dev():
     """Runs the set-up needed for local development."""
     setup_general()
 
+    admin_email = 'wvr@gmail.com'
+    if User.query.filter_by(email=admin_email).first() is None:
```
Comment (sandlerben) [inline manage.py:68]: @jondubin what he currently has is better than `if not User.query.filter_by(email=email).first():`

From PEP8:

> Comparisons to singletons like None should always be done with 'is' or 'is not', never the equality operators.
> 
> Also, beware of writing "if x" when you really mean "if x is not None" -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

Comment (aharelick) [inline manage.py:68]: The Facts

**Pull request merged**